data-access/src/routes/http/graphql/schema/types/member.graphql

```diff
@@ -199,3 +200,30 @@ input AddPaymentInstrumentInput {
   paymentToken: String!
   isDefault: Boolean!
 }
+
+input UpdatePaymentInstrumentInput {
+  cardType: CardType!
+  paymentInstrumentId: String!
+  isDefault: Boolean!
+  expirationYear: String!
+  expirationMonth: String!
+  id: String!
+  billingFirstName: String!
+  billingLastName: String!
+  billingEmail: String!
+  billingPhone: String!
+  billingAddressLine1: String!
+  billingAddressLine2: String
+  billingCity: String!
+  billingState: String!
+  billingPostalCode: String!
+  billingCountry: String!
+}
+
+enum CardType {
+  Visa
+  Master
+  American_Express
+  Discover
+  JCB
```

Comment:
we do not accept JCB, keep the enum limited to Visa, Master Card, Amex and Discover. Also, please verify the accepted values for cybersource.